src/emitter.js

```diff
@@ -214,13 +214,17 @@ CKEDITOR.define( [ 'eventinfo', 'utils' ], function( EventInfo, utils ) {
 		 * @param {...*} [args] Additional arguments to be passed to the callbacks.
 		 */
 		fire: function( event, args ) {
-			var callbacks = getCallbacksIfAny( this, event );
+			var eventInfo = event instanceof EventInfo && event;
```

Comment:
Hum... I thought about that, but I found this a nice feature to have, especially considering that it was already there. I mean, it's a way to fire an event using an EventInfo object (e.g. can be used to re-fire an event from other objects).

Would you prefer me to remove it? Otherwise I'm fine to have tests for it.